src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/oasis/MainPage.java

```diff
@@ -0,0 +1,37 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.oasis;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.contentpatterns.URLsContent'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)